src/entries/popup/pages/messages/SendTransaction/SendTransactionsInfo.tsx

```diff
@@ -437,13 +439,13 @@ function InsuficientGasFunds({
       <Inline alignVertical="center" space="12px">
         <ChainBadge chainId={chainId} size={16} />
         <Text size="14pt" weight="bold">
-          {+nativeAsset.balance.amount > 0
+          {+(nativeAsset?.balance?.amount ?? '0') > 0
```

Comment:
This would be an invalid comparison between a string and number. The `nativeAsset?` should be enough to suffice for the check (no gas when the balance is equal to 0 or nativeAsset is null)

Reply:
i think with type coercion theres no issue in the comparison (?). I can look into the other check though in the morning.